leaguepy/data/retrieve_data.py

```diff
@@ -92,15 +95,122 @@ def get_match_history(
+    #                         - 'attackSpeed'
+    #                         - 'bonusArmorPenPercent'
+    #                         - 'bonusMagicPenPercent'
+    #                         - 'ccReduction'
+    #                         - 'cooldownReduction'
+    #                         - 'health'
+    #                         - 'healthMax'
+    #                         - 'healthRegen'
+    #                         - 'lifesteal'
+    #                         - 'magicPen'
+    #                         - 'magicPenPercent'
+    #                         - 'magicResist'
+    #                         - 'movementSpeed'
+    #                         - 'omnivamp'
+    #                         - 'physicalVamp'
+    #                         - 'power'
+    #                         - 'powerMax'
+    #                         - 'powerRegen'
+    #                         - 'spellVamp'
+    #                     - 'currentGold'
+    #                         - int
+    #                     - 'damageStats'
+    #                         - 'magicDamageDone'
+    #                         - 'magicDamageDoneToChampions'
+    #                         - 'magicDamageTaken'
+    #                         - 'physicalDamageDone'
+    #                         - 'physicalDamageDoneToChampions'
+    #                         - 'physicalDamageTaken'
+    #                         - 'totalDamageDone'
+    #                         - 'totalDamageDoneToChampions'
+    #                         - 'totalDamageTaken'
+    #                         - 'trueDamageDone'
+    #                         - 'trueDamageDoneToChampions'
+    #                         - 'trueDamageTaken'
+    #                     - 'goldPerSecond'
+    #                         - int
+    #                     - 'jungleMinionsKilled'
+    #                         - int
+    #                     - 'level'
+    #                         - int
+    #                     - 'minionsKilled'
+    #                         - int
+    #                     - 'participantId'
+    #                         - int
+    #                     - 'position'
+    #                         - 'x' int
+    #                         - 'y' int
+    #                     - 'timeEnemySpentControlled'
+    #                         - int
+    #                     - 'totalGold'
+    #                         - int
+    #                     - 'xp
+    #                         - int
+    #             - 'timestamp'
+    #                 -int
+    #     - 'gameId'
+    #         - int
+    #     - 'participants'
+    #         - dict[int, str]
+
     URL = URLS["timeline"]
     logger.info(f"Pulling timeline for match_id: {match_id}")
     response = requests.get(URL.format(region=region.value, match_id=match_id), params=RIOT_PARAMS)
     return response
 
 
+def get_player_ids(match: dict[str, dict]) -> set[str]:
+    return set(match.get("metadata", {}).get("participants", {}))
```

Comment:
Won't this be an error? You can't hash a dictionary. Participants are a dict of int -> str (per your comment). So grab the values, if that's the puuids